docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -107,13 +113,20 @@ To enable your application to use the media controller functionality:
 
 - To use the media controller client:
 
-  1. To use the functions and data types of the Media Controller Client API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) applications), include the `<media_controller_client.h>` header file in your application:
+  1. The application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+     ```
+     <privileges>
+        <privilege>http://tizen.org/privilege/mediacontroller.client</privilege>
+     </privileges>
+     ```
+  
+  2. To use the functions and data types of the Media Controller Client API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) applications), include the `<media_controller_client.h>` header file in your application:
```

Comment:
Please remove (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__CONTROLLER__CLIENT__MODULE.html) applications).


```
2. To use the functions and data types of the Media Controller Client API, include the `<media_controller_client.h>` header file in your application:
```

Reply:
Done